cloudnatix/assets/dashboards/cloudnatix_overview.json

```diff
@@ -0,0 +1 @@
height":2}},{"id":5922853403113722,"definition":{"title":"Savings recommended","title_size":"16","title_align":"left","time":{},"type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:cloudnatix.workload.monthly_projected_saving{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":false,"custom_unit":"$/mo","custom_links":[{"label":"Discover insights","link":"https://app.cloudnatix.com/app/insights"}],"precision":2},"layout":{"x":6,"y":0,"width":3,"height":2}},{"id":6190142051270206,"definition":{"title":"instances","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"avg:kubernetes_state.node.count{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2},"layout":{"x":9,"y":0,"width":2,"height":1}},{"id":7654902037777398,"definition":{"title":"cores","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes_state.node.cpu_capacity{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"custom_unit":"cores","precision":2},"layout":{"x":9,"y":1,"width":1,"height":1}},{"id":8825923597966732,"definition":{"title":"memory","title_size":"16","title_align":"left","type":"query_value","requests":[{"response_format":"scalar","queries":[{"name":"query1","data_source":"metrics","query":"sum:kubernetes_state.node.memory_capacity{*}","aggregator":"avg"}],"formulas":[{"formula":"query1"}]}],"autoscale":true,"precision":2},"layout":{"x":10,"y":1,"width":1,"height":1}},{"id":485764349959394,"definition":{"type":"note","content":"[Analyze workloads](https://app.cloudnatix.com/app/workloads)\n\n[Discover insights](https://app.cloudnatix.com/app/insights)\n\n[Review clusters](https://app.cloudnatix.com/app/cluster)","background_color":"white","font_size":"14","text_align":"left","vertical_align":"top","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true},"layout":{"x":0,"y":2,"width":2,"height":2}},{"id":7170024749604466,"definition":{"title":"Workloads","title_size":"16","title_align":"left","type":"query_table","requests":[{"response_format":"scalar","formulas":[{"alias":"autopilot","formula":"query1","limit":{"count":500,"order":"desc"}},{"alias":"manual","formula":"query2"},{"alias":"not applied","formula":"query3"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:autopilot}","aggregator":"avg"},{"name":"query2","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:manual}","aggregator":"avg"},{"name":"query3","data_source":"metrics","query":"sum:cloudnatix.vpa{mode:notapplied}","aggregator":"avg"}]}]},"layout":{"x":2,"y":2,"width":4,"height":2}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
I think I fixed that, thanks for the helpful link.  I added a description panel at the left with a paragraph of explaining what it is and brought a list of links there, and right side have the graphs.  I added a logo image there as well, although https://docs.datadoghq.com/developers/integrations/create-an-integration-dashboard#general-guidelines is still not very clear to me what is the right way to add an image there.  I put a direct link to our image asset in docs.cloudnatix.com domain, if that's not the right idea, please navigate me for how to bring the image there.

However, for the logo, I am still not sure what's the way to do that and what's missing in this PR.  https://docs.datadoghq.com/developers/integrations/create-an-integration-dashboard#general-guidelines this page says about `integration_id`, but there are zero occurrence of this keyword within the entire integrations-extras repository.  But this PR has the logo image within `assets/logos` and the `app_id` and the logo image file name (without `.svg` extension) matches, so I am guessing this would be what the linked guideline is saying, but I'm not particularly sure this is right.

If this is not working as intended, please provide more concrete information for how to add a logo into the dashboard.